controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -449,6 +472,41 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
                 });
     }
 
+    /**
+     * The method translates the internal object ReaderGroupConfigRecord object into REST response object.
+     *
+     * @param scope the scope of the Reader Group.
+     * @param rgName the name of the Reader Group.
+     * @param rgConfig The configuration of Reader Group.
+     * @return Stream properties wrapped in StreamResponse object
+     */
+    private static final ReaderGroupConfiguration encodeReaderGroupConfigRecord(String scope, String rgName,
```

Comment:
should this method be moved to model helper?

Reply:
done.